src/_collections/access/10-remove-audit-editor.md

```diff
@@ -0,0 +1,9 @@
+---
+tags: access
+title: Manage user roles: adding new Audit Editor

```

Comment:
```suggestion
title: "Manage user roles: adding new Audit Editor"
```

Per James P's suggestion, adding quotes should hopefully resolve the issue you're seeing! 

For "frontmatter" (the special stuff between the `---`'s), a colon is a special/reserved character. When the build system comes across one, it just assumes it's looking at a `key: value` pair and gets all confused and thrown off of its orderly process. We can tell it to treat the colon as a "plain" text character by wrapping that text in quotes.